### DISCUSSION TITLE
GetMuxedStreams() does not contain definition for GetWithHighestVideoQuality()?

### DISCUSSION THREAD

**Discussion by ThomasBeHappy** [Tyrrrz/YoutubeExplode#589] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29510664/134646726-7b3dcc77-3e47-4976-a6a6-a9ca140fcbb9.png)

it's telling me
'IEnumerable<MuxedStreamInfo>' does not contain a definition for 'GetWithHighestVideoQuality' and no accessible extension method 'GetWithHighestVideoQuality' accepting a first argument of type 'IEnumerable<MuxedStreamInfo>' could be found (are you missing a using directive or an assembly reference?)

What am I doing wrong here?

**Comment by Tyrrrz** [comment, marked as answer]
Highlight the red underlined code, press `Ctrl+.`, press "Add using statements" or whatever
